README.md

```diff
@@ -1 +1,194 @@
-# .github
\ No newline at end of file
+# **kitexcall**
+
+English | [中文](README_cn.md)
+
+Kitexcall is a command-line tool for sending JSON general requests using kitex, similar to how curl is used for HTTP.
+
+## Features
+
+- **Supports Thrift/Protobuf:** It supports IDL in Thrift/Protobuf formats.
+- **Supports Multiple Transport Protocols:** It supports transport protocols like Buffered, TTHeader, Framed, and TTHeaderFramed, with plans to support GRPC (Protobuf and Thrift Streaming) in the future.
+- **Supports Common Client Options:** It allows specify common client options, such as client.WithHostPorts, etc.
```

Comment:
Format no working. Please review first in your own repo: https://github.com/Zzhiter/kitexcall/tree/add_readme